microbench/src/main/java/io/netty/microbench/buffer/ByteBufIndexOfBenchmark.java

```diff
@@ -74,7 +93,7 @@ public void init() {
         System.setProperty("io.netty.noUnsafe", Boolean.valueOf(noUnsafe).toString());
```

Comment:
It would be nice to add a benchmark version using this last algorithm version but using NIO `ByteBuffer` (array backed and direct memory backed) to see how the new Buffer abstraction compete with it too

Reply:
Both of those are tested through the Buffer API when `noUnsafe=true`, so hypothetically the only difference would be the extra indirection on the initial call.